site/content/en/docs/faq/_index.md

```diff
@@ -159,3 +159,7 @@ $env:MINIKUBE_HOME = "D:\.minikube"
 
 minikube start
 ```
+
+## How can I resolve network issue to download image repository
+
+Simply add flag `--image-repositor` when exec `minikube start`.
```

Comment:
this doesnt seem to be correct flag and I believe it requires passing a value to the `--image-repository`

Reply:
I suggest reword it this way

"Solve "EXACT_ERROR_MESSAGE""

if your company/country blocks access to k8s.io or gcr.io you can use this flag that sets an alternative image repository to pull docker images from. This can be used when you have limited access to gcr.io. Set it to "auto" to let minikube decide one for you. For Chinese mainland users, you may use local gcr.io mirrors such as registry.cn-hangzhou.aliyuncs.com/google_containers
	If set, download tarball of preloaded images if available to improve start time. Defaults to true.